go/src/lib.rs

```diff
@@ -186,8 +189,8 @@ fn create_client_internal(
 ///
 /// * `connection_request_bytes` must point to `connection_request_len` consecutive properly initialized bytes. It must be a well-formed Protobuf `ConnectionRequest` object. The array must be allocated by the caller and subsequently freed by the caller after this function returns.
 /// * `connection_request_len` must not be greater than the length of the connection request bytes array. It must also not be greater than the max value of a signed pointer-sized integer.
-/// * The `conn_ptr` pointer in the returned `ConnectionResponse` must live while the client is open/active and must be explicitly freed by calling [`close_client`].
-/// * The `connection_error_message` pointer in the returned `ConnectionResponse` must live until the returned `ConnectionResponse` pointer is passed to [`free_connection_response`].
+/// * The `conn_ptr` pointer in the returned `ConnectionResponse` must live while the client is open/active and must be explicitly freed by calling [close_client].
```

Comment:
rustdoc adds links for symbols written in `[`func`]`.
Consider reverting this

Reply:
why are the added links undesirable?

Reply:
Sorry, my misundenstating. The link is created by `[]` only, so backticks adjust the style only.
The doc below has `[close_client]` and ``[`free_connection_response`]``. I don't mind the style change, but please ensure it is common across other docs.
![image](https://github.com/user-attachments/assets/0227d19e-4349-485d-ae47-6a290f14a2c8)